packages/shared/src/url.ts

```diff
@@ -37,7 +38,7 @@ const getIpv4Interfaces = () => {
       }
     });
   });
-  return ipv4Interfaces;
+  return uniq(ipv4Interfaces);
```

Comment:
Thanks for your contribution! 😄

I'm wondering that if `lodash.uniq` can be used to uniq objects.

I have written some simple test and it does not work:

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/c56f16c2-197d-497f-87a0-822db82f41ef)

Also Rsbuild will remove `lodash` from dependencies in the future (to reduce dependencies)

So can we use another method to uniq the ipv4Interfaces?

Reply:
Got it,let me see.

Reply:
I use map to uniq address, i'm not sure map is good choose for us.But i think it's better than object